src/traversals/parallel_bfs.jl

```diff
@@ -15,7 +15,7 @@ import Base: push!, shift!, isempty, getindex
 
 export bfs_tree, LevelSynchronousBFS
 
-mutable struct LevelSynchronousBFS <: AbstractGraphVisitAlgorithm end
+struct LevelSynchronousBFS end
```

Comment:
TBH, this is the only threaded implementation of anything in the library, and I'd much rather have a multi-core parallel version.